_src/_posts/2018-01-04-importance-of-serving-ads.markdown

```diff
@@ -124,51 +120,43 @@ These relationships were created from the data, yet we could not explicitly give
 
 As a second step, we needed to define business metrics that are relevant for our customers. We will have that different frequency cappings would produce different levels of them, because of how the relationships \\(Imp\\), \\(Conv\\) and \\(Cl\\) vary along cap levels.
 
-Being a performance marketing platform, we prioritized CPA (cost-per-action) optimization of our clients. At the same time, and given the industry's last-click attribution system, we have that our revenue stream is click dependent. So we took revenue to be a second relevant metric, as measured by clicks. In this way we will have two metrics defined as functions of the previous relationships.
-
-The revenue is very simple in terms of the average CPC (cost-per-click) for that campaign:
-
-$$
-  Rev :  F \rightarrow \mathbb{N} \\
-    f  \rightarrow Cl(f)\times CPC
-  $$
+Being a performance marketing platform, we prioritized CPA (cost-per-action) optimization of our clients. At the same time, and given the industry's last-click attribution system, we have that our revenue stream is click dependent. So we took click volume to be a second relevant metric, measured by the volume of clicks. In this way we will have two metrics defined as functions of the previous relationships.
```

Comment:
"we have that": again, can't use this expression.
Use: last-click attribution system, our revenue stream is click dependent

"So we took": can't start a sentence this way.  
Use: We used click volume as a second relevant metric.

"In this way we will have": can't use "we will have". one option is "we obtain".
Use: In this way, we obtain two metrics defined as functions of the previous relationships.